recipes/tag-mne/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "tag-mne" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tag_mne-{{ version }}.tar.gz
+  sha256: db744a400c67fae4822a10ed6d099aa608aba60e0ab8a601bd0bfd4442f18ae6
+
+build:
+  skip: true
```

Comment:
You are not building anything with this skip here.

Reply:
Thank you, I have made update on it.